package aprendizadoJava.teste.erro.aula48;


class MultiplosCatchTest {

    //@org.junit.jupiter.api.Test
    void main() {
    }
}